packages/dcos-launch/extra/README.md

```diff
@@ -0,0 +1,28 @@
+* Portable shipping: `dcos-launch` is shipped as a frozen python binary so that you never need to be concerned with maintaining an extremely specific develop environment or deployment framework.
+* Build Verification: besides launching clusters, `dcos-launch` can test that the provided cluster is healthy through the `pytest` subcommand. `dcos-launch` is capable of describing the cluster topology and can verify that the expected cluster is present. To learn more about the testing suite triggered by this command, see: [dcos-integration-tests](../../dcos-integration-tests/extra)
+* Programmatic consumption: all information necessary to handle a given deployment created by `dcos-launch` is contained in the generated `cluster_info.json` file. This allows dependent CI tasks to run other tests, describe the cluster, or completely delete the cluster
+
+# Requirements
+* Linux operating system with python3 support
+* SSH client on localhost
+
+## Use
+### `dcos-launch create`
+Consumes a configuration file, performs basic validation on the deployment parameters, and then signals the deployment provider to begin deployment. In the case of third-party provisioning (provider is AWS or Azure), the cluster will eventually finish deploying with no further action. In the case of onprem provisioning, `dcos-launch` needs to partially drive the deployment process, so this command only triggers creation of the underlying bare hosts
+
+### `dcos-launch wait`
+Consumes the cluster info from the create step to see if the deployment is ready. In the case of third-party providers, this command will query their deployment service and surface any errors that may have dereailied the deployment. In the case of onprem provisioning, there are multiple, triggerable stages in deployment, so the wait command *must* be used to complete the deployment
+
+### `dcos-launch describe`
+Consumes the cluster info and outputs the essential parameters of the cluster. E.G. master IPs, agent IPs, load balancer addresses, etc. Additionally, the STDOUT stream is formatted in JSON so that the output can be piped into another process or tools like `jq` can be used to pull out specific paramters
+
+### `dcos-launch pytest`
+Consumes the custer info and runs the `dcos-integration-tests` package. This is the same test suite used to validate DC/OS pull requests. Additionally, arbitrary argumnets and environment variables may be added to the pytest command. For example, if one only wants to run the cluster composition test to see if the cluster is up: `dcos-launch pytest -- test_composition.py`. Anything input after `--` will be injected after `pytest` (which is natually run without options)
+
+### `dcos-launch delete`
+Consums the cluster info and triggers the destruction of the deployment. In cases where dcos-launch provided third-party dependencies via a helper (`zen_helper` or `key_helper` for AWS), `delete` will block until all those resources have been removed
```

Comment:
I reviewed the guide again. I felt, the following details can be added.

1) How to build `dcos-launch` locally or obtain `dcos-launch`?

And you can briefly mention at the end of the guide to invoke `dcos-launch --help` for more details on the the above commands.